code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm

```diff
@@ -62,7 +62,10 @@
 
 /proc/_locate(X, Y, Z)
 	if (isnull(Y)) // Assuming that it's only a single-argument call.
-		return locate(X)
+		// direct ref locate
+		var/datum/D = locate(X)
+		// &&'s to last value
+		return istype(D) && D.can_vv_mark() && D
```

Comment:
```suggestion
		return istype(D) && D.can_vv_mark()
```
optional, but `istype` comes with an implicit null check

Reply:
@MrMelbert the proc returns a datum

&& short circuits to last value, otherwise it'd be

```
if(!istype(D) || !D.can_vv_mark())
  return
return D
```

Reply:
oh what I just realized lol
that's wild